dev/generated/dirty_cat.datasets.fetch_employee_salaries.html

```diff
@@ -52,7 +52,7 @@ <h1><code class="xref py py-mod docutils literal"><span class="pre">dirty_cat.da
 <dt id="dirty_cat.datasets.fetch_employee_salaries">
 <code class="descclassname">dirty_cat.datasets.</code><code class="descname">fetch_employee_salaries</code><span class="sig-paren">(</span><span class="sig-paren">)</span><a class="reference internal" href="../_modules/dirty_cat/datasets/fetching.html#fetch_employee_salaries"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#dirty_cat.datasets.fetch_employee_salaries" title="Permalink to this definition">¶</a></dt>
 <dd><p>fetches the employee_salaries dataset</p>
-<p>the employee_salaries dataset contains information about annual salaries
+<p>The employee_salaries dataset contains information about annual salaries
```

Comment:
This should actually be modified in the Python source of the corresponding function.